.changeset/long-geckos-clap.md

```diff
@@ -2,7 +2,9 @@
 '@swisspost/design-system-styles': patch
 ---
 
+Updated button styles:
+
 - Reduced horizontal padding of tertiary buttons.
 - Updated font-size progression according to button size.
 - Changed transparent color to full-tone colors.
-- Updated slightly the font and border color of secondary and tertiary buttons.
+- Updated the font- and border-color of secondary and tertiary buttons.
```

Comment:
```suggestion
- Updated the font-color and border-color of secondary and tertiary buttons.
```

Reply:
I think it's only in German that you can do that, not sure.